src/ObjectUtils/ElfFile.cpp

```diff
@@ -102,7 +102,7 @@ template <typename ElfT>
 ErrorMessageOr<GnuDebugLinkInfo> ReadGnuDebuglinkSection(
     const typename ElfT::Shdr& section_header, const llvm::object::ELFFile<ElfT>& elf_file) {
   llvm::Expected<llvm::ArrayRef<uint8_t>> contents_or_error =
-      elf_file.getSectionContents(&section_header);
+      elf_file.getSectionContents(section_header);
   if (!contents_or_error) return ErrorMessage("Could not obtain contents.");
```

Comment:
since we do not takeError here for the error message this will actually abort on return.

Reply:
Done.